linux_sql/scripts/host_info.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```

Comment:
I didn't see psql_docker.sh file. please make sure you also merge that into the develop branch for the next PR.